spec/workers/mentions/send_email_notification_worker_spec.rb

```diff
@@ -1,10 +1,15 @@
 require "rails_helper"
 
 RSpec.shared_examples "a valid mentionable" do
+  let(:mailer) { double }
+  let(:message_delivery) { double }
+
   context "with a mention" do
     it "calls on NotifyMailer" do
       worker.perform(mention.id) do
         expect(NotifyMailer).to have_received(:new_mention_email).with(mention)
+        expect(mailer).to have_received(:new_reply_email)
```

Comment:
I'm a little confused - when we're testing the Mentions::SendEmailNotificationWorker - when does NotifyMailer receive new reply email? I do see that in the `Comments::SendEmailNotificationWorker` - I wonder if you meant to include this message or copied one line too many from the other spec?

Reply:
Hey, thanks for this catch! Yeah I think copied too few lines here and didn't have the best understanding of the spec from the comment email worker spec. Should be fixed now 👍